client/src/components/CFP/CFPForm.js

```diff
@@ -66,10 +66,11 @@ class CFPForm extends Component {
   }
 
   setValue = _.debounce((form, key, value) => {
+    const currentRelevantForm = _.get(this.state, form)
+    const updatedRelevantForm = _.assign({}, currentRelevantForm, { [key]: value })
```

Comment:
1. read about lodash assign
2. it can be done also with spread operator, but I like this more, personal thing
3. the idea is to create a new object only with the relevant change inside
4. what we had here before was overriding the entire form object only with the key coming from the event handler
5. so when the component rendered below, it tried to extract other props from state that were missing